### ISSUE TITLE
Bulk Upload Fails Silently

### ISSUE THREAD

**Issue by gordonbanderson** [manticoresoftware/manticoresearch-php#9]
hi,

I was trying to write a bulk upload test, I went with this initially:

https://github.com/gordonbanderson/manticoresearch-php/blob/testing/test/Manticoresearch/Endpoints/BulkTest.php#L24

Now https://github.com/gordonbanderson/manticoresearch-php/blob/testing/test/Manticoresearch/Endpoints/BulkTest.php#L24 is erroneous in that a product with id of 100 already exists, an oversight on my part.  However https://github.com/gordonbanderson/manticoresearch-php/blob/testing/test/Manticoresearch/Endpoints/BulkTest.php#L32 does not fail.  If I add an error_log statement in src/Manticoresearch/Response.php the id duplication error shows in the response returned by the Manticore server.

I think likes of a RuntimeException should be thrown

Regards

Gordon

**Comment by gordonbanderson** [comment]
The response is this

```
[RESPONSE: ]
[Array
(
    [items] => Array
        (
            [0] => Array
                (
                    [insert] => Array
                        (
                            [error] => Array
                                (
                                    [type] => duplicate id '100'
                                    [index] => products
                                )

                            [status] => 500
                        )

                )

        )

    [errors] => 1
)
]
```
The existence of `[errors] => 1` ought to trigger an exception at the PHP level in my opinion

**Comment by adriannuta** [comment]
This was fixed at 1e133abc3900e443b3d37b6ef236c2e682d07d66,
